### PR TITLE
[CI:DOCS] Add note about QUADLET_UNIT_DIRS to simplify quadlet debug

### PR DESCRIPTION
**Pull request by ygalblum**
#### Does this PR introduce a user-facing change?
No

```release-note
None
```

### CODE REVIEW

File: docs/source/markdown/podman-systemd.unit.5.md [hunk 1]
```diff
@@ -83,6 +83,13 @@ View the generated files and/or error messages with:
 /usr/lib/systemd/system-generators/podman-system-generator {--user} --dryrun
 ```
 
+#### Debugging a limited set of unit files
+
+If you would like to debug a limited set of unit files, you can copy them to a separate directory and set the
+`QUADLET_UNIT_DIRS` environment variable to this directory when running the command below.
```
Comment (vrothberg) [inline docs/source/markdown/podman-systemd.unit.5.md:89]: > the command below.

I am not sure which command is being referred to.

Comment (ygalblum) [reply docs/source/markdown/podman-systemd.unit.5.md]: You're right. When I first wrote it I meant above but wrote below. I've kept below and added the command. I hope this is better

**Review by vrothberg — APPROVED** [review]
LGTM

**Review by Luap99 — APPROVED** [review]
/lgtm

**Pull request merged**